### PR TITLE
implemented GET for retieving qrcode png from supabase bucket

### PR DESCRIPTION
**Pull request by kssampson**
- Provides a url that leads directly to the image

### CODE REVIEW

**Comment by kssampson** [comment]
@bzw24 , included you in the event you want something changed to use in an email. This should be a working solution as you render the image on your page?

File: server/controllers/getQrController.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const supabase = require('../db');
+
+const getQrCode = async (reservationId) => {
+
+  try {
+    const filePath = `${reservationId}.png`;
+    const { data, error } = await supabase.storage.from('qrcodes').getPublicUrl(filePath);
```
Comment (justcord10) [inline server/controllers/getQrController.js:7]: Query looks good to get the qr png from the bucket!

File: server/routes/getQrRouter.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+const getQrRouter = require('express').Router();
+const { getQrCode } = require('../controllers/getQrController');
+
+getQrRouter
+  .get('/:reservationId', async (req, res) => {
+    try {
+      console.log('req.query.reservationId', req.params.reservationId)
```
Comment (justcord10) [inline server/routes/getQrRouter.js:7]: at some point make sure to remove this, fine for now

**Review by justcord10 — APPROVED** [review]
query looks good, router and controller looks good as well! there is one console log that should be removed (in your router on line 7), but that can be done later. good job!

**Review by kjmak14 — APPROVED** [review]
Nice work! Looking forward to implementing this for the front end

**Review by kjmak14 — APPROVED** [review]
Thanks for resolving those conflicts

**Pull request merged**